### PR TITLE
fix: pass verbose to evaluators

### LINKED ISSUE #2571: [ENHANCEMENT] implement verbose=True in run_evals()
**Is your feature request related to a problem? Please describe.**
run_evals() ignores verbose

**Describe the solution you'd like**
run_evals(...verbose=True) prints LLM reposes to console

**Describe alternatives you've considered**
I'd rather only step through via debug or hijack invocations on LLM level

**Additional context**
https://github.com/Arize-ai/phoenix/blob/main/packages/phoenix-evals/src/phoenix/evals/classify.py#L219

### PR DESCRIPTION
**Pull request by mkhludnev**
fix #2571

### CODE REVIEW

**Review by anticorrelator — APPROVED** [review]
Thank you so much for fixing this!

File: packages/phoenix-evals/src/phoenix/evals/classify.py [hunk 1]
```diff
@@ -271,6 +271,7 @@ async def _arun_eval(
             payload.record,
             provide_explanation=provide_explanation,
             use_function_calling_if_available=use_function_calling_if_available,
+            verbose=verbose
```
Comment (anticorrelator) [inline packages/phoenix-evals/src/phoenix/evals/classify.py:274]: missing a comma

File: packages/phoenix-evals/src/phoenix/evals/classify.py [hunk 2]
```diff
@@ -280,6 +281,7 @@ def _run_eval(
...
```
Comment (anticorrelator) [inline packages/phoenix-evals/src/phoenix/evals/classify.py:284]: same as above

**Comment by axiomofjoy** [comment]
@mkhludnev Thanks for the fix! It looks like this PR is failing formatting in CI. Can you fix manually by adding trailing commas or run `hatch run style:fix`?

File: packages/phoenix-evals/src/phoenix/evals/classify.py (hunk 1 above)
Comment (mikeldking) [inline packages/phoenix-evals/src/phoenix/evals/classify.py:274]:
```suggestion
            verbose=verbose,
```

File: packages/phoenix-evals/src/phoenix/evals/classify.py (hunk 2 above)
Comment (mikeldking) [inline packages/phoenix-evals/src/phoenix/evals/classify.py:284]:
```suggestion
            verbose=verbose,
```

**Comment by mkhludnev** [comment]
Pardon, I stepped for a while. Thanks for fixing commas and alignment. 
I launched `test_classify.test_run_evals_outputs_dataframes_with_just_labels_and_scores` with `verbose=True` it prints to console exactly what I need. 
Is it worth to add test case asserting console output, or this may go as is?

**Comment by mikeldking** [comment]
> Pardon, I stepped for a while. Thanks for fixing commas and alignment. I launched `test_classify.test_run_evals_outputs_dataframes_with_just_labels_and_scores` with `verbose=True` it prints to console exactly what I need. Is it worth to add test case asserting console output, or this may go as is?

Good to go! Thanks again for contributing

**Pull request merged**